packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -1545,6 +1546,99 @@ curl -v -X POST \
 }
 ```
 
+#### Enroll WebAuthn Factor
+
+Enrolls a user with a WebAuthn factor. The enrollment process starts with getting the WebAuthn credential creation options that are used to help select an appropriate authenticator using the WebAuthn API.
+This authenticator then generates an enrollment attestation, which may be used to register the authenticator for the user.
+
+##### Enroll WebAuthn Response Parameters
+
+In the [embedded resources](#embedded-resources) object, the `response._embedded.activation` object contains properties used to guide the client in creating a new WebAuthn credential for use with Okta.
+
+For more information about these credential creation options, see the [WebAuthn spec for PublicKeyCredentialCreationOptions](https://www.w3.org/TR/webauthn/#dictionary-makecredentialoptions).
+
+##### Enroll WebAuthn Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "factorType": "webauthn",
+  "provider": "FIDO"
+}' "https://${yourOktaDomain}/api/v1/users/00u15s1KDETTQMQYABRL/factors"
+```
+
+##### Enroll WebAuthn Response Example
+
+```json
+{
+  "id":"fwf2rovRxogXJ0nDy0g4",
+  "factorType":"webauthn",
+  "provider":"FIDO",
+  "vendorName":"FIDO",
+  "status":"PENDING_ACTIVATION",
+  "created":"2018-05-24T20:43:19.000Z",
+  "lastUpdated":"2018-05-24T20:43:19.000Z",
+  "_links":{
+    "activate":{
+      "href":"https://${yourOktaDomain}/api/v1/users/00u15s1KDETTQMQYABRL/factors/fwf2rovRxogXJ0nDy0g4/lifecycle/activate",
+      "hints":{
+        "allow":[
+          "POST"
+        ]
+      }
+    },
+    "self":{
+      "href":"https://${yourOktaDomain}/api/v1/users/00u15s1KDETTQMQYABRL/factors/fwf2rovRxogXJ0nDy0g4",
+      "hints":{
+        "allow":[
+          "GET"
+        ]
+      }
+    },
+    "user":{
+      "href":"https://${yourOktaDomain}/api/v1/users/00u15s1KDETTQMQYABRL",
+      "hints":{
+        "allow":[
+          "GET"
+        ]
+      }
+    }
+  },
+  "_embedded":{
+    "activation": {
+      "attestation": "direct",
+      "authenticatorSelection": {
+        "userVerification": "preferred",
+        "requireResidentKey": false
+      },
+      "challenge": "cdsZ1V10E0BGE4GcG3IK",
+      "excludeCredentials": [],
+      "pubKeyCredParams": [
+        {
+          "type": "public-key",
+          "alg": -7
+        },
+        {
+          "type": "public-key",
+          "alg": -257
+        }
+      ],
+      "rp": {
+        "name":"Rain-Cloud59"
+      },
+      "user": {
+        "displayName": "First Last",
+        "name": "first.last@gmail.com",
+        "id": "00u15s1KDETTQMQYABRL"
+      }
+    }
+  }
+}
+```
+
 #### Enroll Custom HOTP Factor
 
```

Comment:
While you are here, can you also rectify the sample for HOTP as well.

There is a comma missing after  `"factorProfileId": "fpr20l2mDyaUGWGCa0g4"` a few lines below.